.yarn/versions/a282bd58.yml

```diff
@@ -0,0 +1,36 @@
+releases:
+  "@yarnpkg/fslib": patch
```

Comment:
A few more packages need to be released:
- `@yarnpkg/pnp`, `@yarnpkg/plugin-pnp` for the PnP hook (and `yarn update:pnp:hook` needs to be run too)
- `vscode-zipfs` because it uses the `ZipOpenFS`
- `@yarnpkg/cli` because the hook changes affect the CLI
(I hope I didn't miss any)

Reply:
All the packages in `declined` depend on `@yarnpkg/fslib`, how can I decide which packages need to be released?

Reply:
Since it's a patch you can leave them all as declined except the ones Paul mentioned (because those are bundling @yarnpkg/fslib, so they won't get the new version unless they are re-released).